Get-Scope.ps1

```diff
@@ -0,0 +1,29 @@
+function Get-Scope-Array {
+    param(
+        [string] $currentRelativePath
+    )
+
+    if( $(Split-Path $currentRelativePath -Parent) -ne '.' ) {
+        Get-Scope-Array -currentRelativePath $(Split-Path $currentRelativePath)        
+    }
+
+    $scope = $(Split-Path -Leaf $currentRelativePath).Split('.')[0]
+    if(-not $scope.Contains('device')) {
+        $scope
+    }    
+}
+
+Set-Location C:\Work\Git\Fleetbot\deploy
```

Comment:
Not sure if we should use a hard coded path here. Maybe you can try 
`Set-Location "$(git rev-parse --show-toplevel)\deploy" ` ?

Reply:
Agree, I updated in the next iteration